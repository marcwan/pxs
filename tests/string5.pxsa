        ADDVAR          asdfasdfasdf_zxcvz_xcvxv
        MUL             "12"       , "10"           , asdfasdfasdf_zxcvz_xcvxv
        DEBUGPRINT      asdfasdfasdf_zxcvz_xcvxv
MY_SUPER_AWESOME_LABEL:
        EXIT            0
