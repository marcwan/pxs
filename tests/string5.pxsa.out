DEBUGPRINT: asdfasdfasdf_zxcvz_xcvxv = 120.000000
